### PR TITLE
Add pixel gpu vs cpu plots to shift workspace

### PR DESCRIPTION
**Pull request by sroychow**
This PR adds plots to the PixelPhase1 shift workspace for online GUI. The plots are for pixel GPU vs CPU comparisons.
Once merged, instructions for online shifters to follow. 

Would request a playback test on a pp collision run.

### CODE REVIEW

**Comment by nothingface0** [comment]
This PR has been tested on our online playback machines. It can be deployed in production whenever you prefer.

**Comment by sroychow** [comment]
@nothingface0 can you re-test this please?

**Comment by nothingface0** [comment]
@sroychow It's been redeployed, go ahead and take a look: https://cmsweb.cern.ch/dqm/online-playback/

File: dqmgui/layouts/shift_pixelPhase1_layout.py [hunk 1]
```diff
@@ -90,3 +90,79 @@ def shiftpixelP1layout(i, p, *rows): i["00 Shift/PixelPhase1/" + p] = DQMItem(la
...
+      'draw': { 'withref': "no", 'drawopts': "COLZ", 'ytype': 'log' }},
+    { 'path': "SiPixelHeterogeneous/PixelRecHitsCompareGPUvsCPU/rechitsposXDiffFpix",
+      'description': "x-position difference in Forward",
+      'draw': { 'withref': "no", , 'drawopts': "COLZ", 'ytype': 'log' }}],
+   [{ 'path': "SiPixelHeterogeneous/PixelRecHitsCompareGPUvsCPU/rechitsposYDiffBpix",
+      'description': "y-position difference in Barrel",
+      'draw': { 'withref': "no", 'drawopts': "COLZ", 'ytype': 'log' }},
+    { 'path': "SiPixelHeterogeneous/PixelRecHitsCompareGPUvsCPU/rechitsposYDiffFpix",
+      'description': "y-position difference in Forward",
+      'draw': { 'withref': "no", , 'drawopts': "COLZ", 'ytype': 'log' }}]
```
Comment (nothingface0) [inline dqmgui/layouts/shift_pixelPhase1_layout.py:152]:
```suggestion
      'draw': { 'withref': "no", 'drawopts': "COLZ", 'ytype': 'log' }}]
```

File: dqmgui/layouts/shift_pixelPhase1_layout.py [hunk 2]
```diff
@@ -90,3 +90,79 @@ def shiftpixelP1layout(i, p, *rows): i["00 Shift/PixelPhase1/" + p] = DQMItem(la
...
+      'draw': { 'withref': "no", 'drawopts': "COLZ", 'ztype': 'log' }}]
+   )
+
+shiftpixelP1layout(dqmitems, "11 - Difference in rechit position",
+   [{ 'path': "SiPixelHeterogeneous/PixelRecHitsCompareGPUvsCPU/rechitsposXDiffBpix",
+      'description': "x-position difference in Barrel",
+      'draw': { 'withref': "no", 'drawopts': "COLZ", 'ytype': 'log' }},
+    { 'path': "SiPixelHeterogeneous/PixelRecHitsCompareGPUvsCPU/rechitsposXDiffFpix",
+      'description': "x-position difference in Forward",
+      'draw': { 'withref': "no", , 'drawopts': "COLZ", 'ytype': 'log' }}],
```
Comment (nothingface0) [inline dqmgui/layouts/shift_pixelPhase1_layout.py:146]:
```suggestion
      'draw': { 'withref': "no", 'drawopts': "COLZ", 'ytype': 'log' }}],
```

**Comment by nothingface0** [comment]
@sroychow redeployed

**Comment by sroychow** [comment]
The shift layout after deploying on online playback looks good. This can deployed in production. 
Thank you @nothingface0

**Comment by sroychow** [comment]
image

For reference.

**Comment by nothingface0** [comment]
This PR has now been deployed in production (online)!

**Pull request merged**